api/src/utils/Logger.ts

```diff
@@ -0,0 +1,19 @@
+import winston from "winston";
+import { environment } from "..";
+
+const Logger = winston.createLogger({
+  level: environment === "development" ? "debug" : "warn",
+  format: winston.format.combine(
+    winston.format.timestamp({ format: "YYYY-MM-DD HH:mm:ss:ms" }),
```

Comment:
W: Use ISO 8601 string, to allow for logs to be easier to be parsed by another program.

Reply:
After digging through the code, I've removed the format option as it should default to the ISO standard. Commit: 7c4ade60202b76ef0680398684bc624fe3d2fde3